analyses/tp53_nf1_score/01-apply-classifier.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Apply Machine Learning Classifiers to PDX RNAseq data
+# 
+# **Gregory Way, 2018**
+# 
+# In the following notebook, I apply three distinct classifiers to patient derived xenograft (PDX) RNAseq data (FPKM).
```

Comment:
It would be helpful to update these comments so that they apply to the current analysis.

To cite the previous code, you can add something along the lines of:

"Modified from PDX PPTC Machine Learning Analysis" and then add the github link and paper URL

Reply:
Some of the comments still apply, so feel free to pick and choose. The code for the other analysis is under an open source license so you are free to modify! 🎉